cmd/node/config/config.toml

```diff
@@ -800,6 +800,11 @@
     [Debug.EpochStart]
         GoRoutineAnalyserEnabled = true
         ProcessDataTrieOnCommitEpoch = true
+    [Debug.Process]
+        Enabled = true
+        LogLevelChanger = "*:DEBUG,p2p:TRACE,debug:DEBUG,process:TRACE,intercept:TRACE"
```

Comment:
`LogLevelChanger` I think this can be renamed 
---> `DebuggingLogLevel` or ` NewLogLevel`

Reply:
done